packages/ui/src/domainModules/collectionMammals/components/MammalManager/MainColumn/RecordForm/formParts/RecordHistoryEvents/EventRow.js

```diff
@@ -0,0 +1,33 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { Grid } from 'semantic-ui-react'
+
+import { getYMDHMSFromTimestamp } from 'common/es5/date'
+
+const propTypes = {
+  actionType: PropTypes.string.isRequired,
+  timestamp: PropTypes.string.isRequired,
+  userName: PropTypes.string,
+}
+
+const defaultProps = {
+  userName: '',
```

Comment:
unless there is a special reason, we use `undefined` as default prop and in this case there is no reason why it would have to be a string